recipes/libevent/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "2.0.22" %}
+
+package:
+  name: libevent
+  version: {{ version }}
+
+source:
+  fn: libevent-{{ version }}.tar.gz
+  url: https://github.com/libevent/libevent/archive/release-{{ version }}-stable.tar.gz
+  sha256: ab89639b0819befb1d8b293d52047c6955f8d1c9150c2b22a0e6247930eb9128
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - toolchain
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - openssl 1.0.*
+    - python
+
+  run:
+    - openssl 1.0.*
+    - python
```

Comment:
Opened issue ( https://github.com/ContinuumIO/anaconda-recipes/issues/44 ) upstream about this problem.